src/appengine/libs/issue_management/issue_tracker.py

```diff
@@ -273,6 +273,10 @@ def label_type(self):
     """Label type."""
     return 'label'  # default
```

Comment:
seems like label_type is no longer needed, so we can remove to keep things simple?

Reply:
It is used in one place here - https://github.com/google/clusterfuzz/blob/feff2e0914ae136291d85e01e43257d93a179c84/src/appengine/handlers/cron/cleanup.py#L1054
We can remove it if we fix this case somehow, but i don't see this as a big deal since we have a better label_type for google issue tracker (hotlist vs hotlistid) for this case.